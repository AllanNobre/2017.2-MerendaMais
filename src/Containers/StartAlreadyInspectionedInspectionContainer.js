import { connect } from 'react-redux';
import StartAlreadyInspectionedInspection from '../screens/StartAlreadyInspectionedInspection';

const mapStateToProps = state => ({
  counselor: state.counselor,
  listOfAlreadyInpectionedSchedueInAGroup:
   state.list.listOfAlreadyInpectionedSchedueInAGroup,
});

const StartAlreadyInspectionedInspectionContainer =
 connect(mapStateToProps)(StartAlreadyInspectionedInspection);

export default StartAlreadyInspectionedInspectionContainer;
