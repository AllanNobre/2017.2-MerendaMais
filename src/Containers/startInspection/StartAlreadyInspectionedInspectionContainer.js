import { connect } from 'react-redux';
import StartAlreadyInspectionedInspection from '../../screens/startInspection/StartAlreadyInspectionedInspection';

const mapStateToProps = state => (
  {
    isLoading: state.application.isLoading,
    counselor: state.counselor,
    listOfAlreadyInpectionedSchedueInAGroup:
   state.list.listOfAlreadyInpectionedSchedueInAGroup,
  }
);

const StartAlreadyInspectionedInspectionContainer =
 connect(mapStateToProps)(StartAlreadyInspectionedInspection);

export default StartAlreadyInspectionedInspectionContainer;
