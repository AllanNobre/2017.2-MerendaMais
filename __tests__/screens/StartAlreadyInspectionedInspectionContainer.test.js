import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import configureStore from 'redux-mock-store';
// import configureStore from 'redux-mock-store';
import StartAlreadyInspectionedInspectionContainer from '../../src/Containers/StartAlreadyInspectionedInspectionContainer';

Enzyme.configure({ adapter: new Adapter() });

const mockStore = configureStore();

const initialState = {

  list: {
    listOfExpiredScheduleInAGroup: {
      codSchool: '',
      date: '',
      time: '',
    },
  },
};

const store = mockStore(initialState);

describe('Testing StartPendingInspection Screen', () => {
  it('renders as expected', () => {
    const wrapper = shallow(
      <StartAlreadyInspectionedInspectionContainer />,
      { context: { store } },
    );
    expect(wrapper).toMatchSnapshot();
  });
});
