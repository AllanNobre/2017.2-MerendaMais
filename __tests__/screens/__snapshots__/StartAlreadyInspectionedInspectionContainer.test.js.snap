// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing StartPendingInspection Screen renders as expected 1`] = `
<StartAlreadyInspectionedInspection
  dispatch={[Function]}
  isLoading={false}
/>
`;
